stdlib/public/Platform/ucrt.swift

```diff
@@ -13,6 +13,11 @@
 @_exported import ucrt // Clang module
 // Extra clang module that's split out from ucrt:
 @_exported import _complex
+@_exported import _fenv
+@_exported import _float
+@_exported import _malloc
+@_exported import _stddef
+@_exported import _stdlib
```

Comment:
This makes me believe that an `import ucrt` is now not going to do what it did before. There were many users where `ucrt` was being imported directly to get access to these modules. Can we do something to not break those clients?